ommprotocol/md.py

```diff
@@ -530,8 +530,8 @@ def restraint_force(self, indices=None, strength=5.0):
         positions = self.positions if self.positions is not None else self.handler.positions
         if indices is None:
             indices = range(self.handler.topology.getNumAtoms())
-        for i, index in enumerate(indices):
-            force.addParticle(i, positions[index].value_in_unit(u.nanometers))
+        for index in indices:
+            force.addParticle(int(index), positions[index].value_in_unit(nanometers))
```

Comment:
Why do we need a explicit `int` cast here? I think the indices are returned as `int` already (otherwise we wouldn't be able to use them as indices in `positions`). Also, we do need the `u.` prefix for `nanometers` (it's an alias to `simtk.units`).